hazelcast/src/main/java/com/hazelcast/internal/management/events/ReloadEvent.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.management.events;
+
+/**
+ * Marker interface for reload events.
+ */
+public interface ReloadEvent {
```

Comment:
I only see usages of this marker interface in a test on EE-side PR. Is introducing this interface generally useful or can we skip adding it?

Reply:
It's not needed for MC too, we rely on `EventMetadata.EventType` to determine event type

Reply:
https://github.com/hazelcast/hazelcast/pull/20260/commits/6b5ae520dfd83a99386559b18eb8546f7e6b75e8